### DISCUSSION TITLE
Animations when switching between MacOS spaces

### DISCUSSION THREAD

**Discussion by hakonmuggerud** [FelixKratz/SketchyBar#413] [Q&A]
I have a setup where I'm quickly changing between many different desktop spaces. The default animations from MacOS are a bit excessive so I have turned on the "reduce motion" accessibility setting. This replaces the animation with a fade instead. 

The issue I'm having is that the SketchyBar is hidden during the fade animation. I've seen some examples of configs where people are using animations in the bar so I'm thinking it must be possible to configure this so that the bar is always visible during the switches. 

Im on Ventura 13.4 currently. Does anyone have any experience with this?

[video example](https://imgur.com/a/9EH3MSh)

**Comment by FelixKratz** [comment]
You could try to use the `sketchybar --bar sticky=on` property.

**Comment by hakonmuggerud** [comment]
Thank you for the suggestion, I couldn't quite get it to work with the native switching in MacOS. Now I'm using yabai space switching as you suggested [here](https://github.com/FelixKratz/SketchyBar/issues/220), and its working properly now
